exercises/practice/rectangles/.meta/gen.go

```diff
@@ -44,14 +37,12 @@ var testCases = []struct {
 	description	string
 	input       []string
 	expected	  int
-}{ {{range .J.Cases}}
-{
+}{
+{{range .J.rectangles}}{
 	description:	{{printf "%q"  .Description}},
-	input:		[]string{
-		{{range .Input.Strings}}{{printf "%q" .}},
-		{{end}}
-	},
-	expected:	{{printf "%d"  .Expected}},
-},{{end}}
+	input:			{{printf "%#v"  .Input.Strings}},
```

Comment:
The old way of writing the input was much more readable in the context of the exercise as you could see the rectangles. I would recommend to go back to that version.

Reply:
true